.github/workflows/prepare-release.yml

```diff
@@ -26,7 +26,8 @@ jobs:
       run: |
         echo "Current version: $CURRENT_VERSION"
         echo "New version: $NEW_VERSION"
-        find . -type f -not -path '*/\.*' -not -path './CHANGELOG.md' -not -path './docs/package-lock.json' -not -path './RELEASING.md' -exec sed -i "s/$CURRENT_VERSION/$NEW_VERSION/g" {} +
+        SCAPED_CURRENT_VERSION=$(echo $CURRENT_VERSION | sed 's/\./\\./g')
```

Comment:
Using 0.8.1 without scaping the dots, were updating 0x8x1 as well, and this affected a class hash. Scaping the dots fix the issue.

Reply:
It was replacing this: 
0x023e416842ca96b1f7067693892ed`00881`d97a4b0d9a4c793b75cb887944d98d
by
0x023e416842ca96b1f7067693892ed`0.9.0`d97a4b0d9a4c793b75cb887944d98d

Reply:
Ahh I understand. Thank you, sir